src/main/java/hudson/plugins/copyartifact/CopyArtifact.java

```diff
@@ -412,7 +426,7 @@ public void perform(@Nonnull Run<?, ?> build, @Nonnull FilePath workspace, @Nonn
                 throw new AbortException(message);
             }
         }
-        FilePath targetDir = workspace, baseTargetDir = targetDir;
```

Comment:
Note for me: `baseTargerDir` looks not used at all. Completely useless.

Reply:
Right, AFAICT it was never used anyway.